src/modules/verifiable-credentials/verifiable-credentials-base.service.ts

```diff
@@ -93,12 +101,43 @@ export abstract class VerifiableCredentialsServiceBase {
     proof_options: string
   ): Promise<string>;
 
-  constructor(protected readonly _signerService: SignerService) {}
+  constructor(
+    protected readonly _signerService: SignerService,
+    private readonly _storage: VCStorageClient
+  ) {}
+
+  /**
+   * @description requests verifiable credentials of type `type`. Only EWF credentials currently supported
```

Comment:
The type that we're currently supporting is `vc-api-exchange`. This is independent from EWF credentials.
```suggestion
   * @description The type of the exchange. Only vc-api exchanges currently supported. 
```